tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -66,8 +66,13 @@ function configure_cri_runtime() {
 		configure_containerd
 		;;
 	esac
-	systemctl daemon-reload
-	systemctl restart "$1"
+	
+	if [ -x /sbin/openrc-run ]; then
+		service k3s-service restart
+	else
+		systemctl daemon-reload
+		systemctl restart "$1"
+	fi
```

Comment:
I'd create a new function, something like:
```sh
function restart_service() {
    if [ -x /sbin/openrc-run ]; then
        service "$1" restart
    else
        systemctl daemon-reload
        systemctl restart "$1"
    fi
}
```

And then simply call:
```sh
    restart_service "$1"
```

Reply:
Good point